molecule_doc.py

```diff
@@ -0,0 +1,78 @@
+"""
+Reads molecule inventory files with the following structure:
+### Tested Functionality 1
+### Test Functionality 2
+### etc...
+
+And writes to formatted markdown file.
+"""
+
+import os
+
+# Set path to molecule scenarios
+path = 'roles/confluent.test/molecule/'
```

Comment:
Can we have a CLI argument mentioning the branch and let the python script resolve the path using that. 
Something like, path resolving to `molecule/ `for `python doc.py 7.0.x `

Reply:
Why?  The script should be specific to the branch you are on (either pre or post 7.0), and it will never change location again.  The different location is only due to restructuring of the project, which is not a regular occurrence.

Reply:
Yeah, ok. I was just bouncing a thought that came to my mind.